src/build/__main__.py

```diff
@@ -78,8 +78,8 @@ def _error(msg: str, code: int = 1) -> NoReturn:  # pragma: no cover
     :param msg: Error message
     :param code: Error code
     """
-    print('{red}ERROR{reset} {}'.format(msg, **_STYLES))
-    exit(code)
+    print('{red}ERROR{reset} {}'.format(msg, **_STYLES), file=sys.stderr)
+    raise SystemExit(code)
```

Comment:
`exit()` is not part of the language, it's injected by `site.py`. Better to use `sys.exit()`, or even better, just `raise SystemExit` directly.

Reply:
Yes, that is a bug.